apps/passport-server/src/routing/routes/statusRoutes.ts

```diff
@@ -9,9 +9,9 @@ export function initStatusRoutes(
   { dbPool }: ApplicationContext,
   {
     semaphoreService,
-    pretixSyncService,
+    zuzaluPretixSyncService: pretixSyncService,
```

Comment:
if you use the auto-renamer in vs-code sometimes you get left with these dangling names